roles/kubernetes-apps/scheduler_plugins/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+scheduler_plugins_controller_replicas: 1
+
+scheduler_plugins_list:
+  - CoScheduling
+  - CapacityScheduling
+  - NodeResources
+
+scheduler_configuration_path: "{{ kube_config_dir }}/scheduler-config.yml"
+
+scheduler_plugins_enabled: false
+
+scheduler_plugin: "CoScheduling"
```

Comment:
is this a single choice ?

Reply:
It's a single choice so far, and allowing multi-choices is more flexible indeed. But I have some hesitation as to whether this version simply introduces scheduler-plugins or provides more complete and flexible support for the it.